pkg/operations/kubernetesupgrade/upgradecluster.go

```diff
@@ -261,17 +262,17 @@ func WriteTemplate(
 
 	templateapp, err := acsengine.PrettyPrintArmTemplate(string(updatedTemplateJSON))
 	if err != nil {
-		log.Fatalf("error pretty printing template: %s \n", err.Error())
+		logrus.Fatalf("error pretty printing template: %s \n", err.Error())
```

Comment:
This is scary to me. Can we return err instead of killing the process/container/app?